products/angular.md

```diff
@@ -31,6 +31,7 @@ releases:
     eol: 2024-05-18
     latest: "15.2.9"
     releaseDate: 2022-11-16
+    lts: 2023-05-03
```

Comment:
Based on the Active Support end date at https://angular.io/guide/releases#actively-supported-versions

```suggestion
    lts: 2023-05-18
```

Reply:
It shows 2023-05-03 under "active". What am I missing?
![Screenshot_20230614_102908_Samsung Internet.jpg](https://github.com/endoflife-date/endoflife.date/assets/1831465/87c12f6c-6cf0-43ad-ad63-eadbba0a9b8b)

Reply:
Sorry about that, must have misread.